src/module/vector.jl

```diff
@@ -138,17 +138,33 @@ function *(a::svector{spoly{T}}, b::spoly{T}) where T <: AbstractAlgebra.RingEle
    return svector{spoly{T}}(R, a.rank, s)
 end
 
+function *(a::svector{Singular.spluralg{T}}, b::Singular.spluralg{T}) where T <: AbstractAlgebra.RingElem
+   base_ring(a) != parent(b) && error("Incompatible base rings")
+   R = base_ring(a)
+   s = GC.@preserve a b R libSingular.pp_Mult_qq(a.ptr, b.ptr, R.ptr)
```

Comment:
Is `libSingular.pp_Mult_qq` the correct function to call here? Maybe @hannes14 can comment